index.html

```diff
@@ -1,182 +1,265 @@
 
+          <ul>
+            <li class="html-btn"><img src="media/html.png" alt="html" /></li>
+            <li class="html-btn"><img src="media/css.png" alt="css" /></li>
+            <li class="html-btn">
+              <img src="media/Js.png" alt="javaScript" />
+            </li>
+          </ul>
+        </div>
+        <button type="button" class="big-btn">See Project</button>
+      </div>
+    </div>
+    <div class="intro">
+      <h2>About</h2>
+      <h2>Myself</h2>
+      <p>
+        I'm a software developer and I can help you build a product, feature or
+        website, look through my work and experience! If you like what you see
+        and have a project you need coded don't hesitate to contact me.
+      </p>
+      <br />
+      <a href="#">LET'S CONNECT</a>
+      <br />
+      <br />
+      <ul class="ul">
+        <li>
+          <a href="#"><img src="media/github.svg" alt="github" /></a>
+        </li>
+        <li>
+          <a href="#"><img src="media/Linkedin.svg" alt="Linkedin" /></a>
+        </li>
+        <li>
+          <a href="#"><img src="media/hand.svg" alt="icon" /></a>
+        </li>
+        <li>
+          <a href="#"><img src="media/twitter.svg" alt="twitter" /></a>
+        </li>
+        <li>
+          <a href="#"><img src="media/M.svg" alt="icon" /></a>
+        </li>
+      </ul>
+      <br />
+      <button type="button" id="resume">Get my resume</button>
+      <div class="arrow">
+        <h3>Languages</h3>
+        <span class="fas fa-angle-down"></span>
+      </div>
+
+      <br />
+      <br />
+      <ul id="languages">
+        <li>
+          <a href="#"><img src="media/Js.svg" alt="javaScript logo" /></a>
+        </li>
+        <li>
+          <a href="#"><img src="media/html5.svg" alt="html5 logo" /></a>
+        </li>
+        <li>
+          <a href="#"><img src="media/css3.svg" alt="css3 logo" /></a>
+        </li>
+      </ul>
+      <br />
+      <div class="border arrow">
+        <h2>Frameworks</h2>
+        <span class="fas fa-angle-right"></span>
+      </div>
+      <div class="border arrow">
+        <h2>Skills</h2>
+        <span class="fas fa-angle-right"></span>
+      </div>
+    </div>
+    <div class="form-section">
+     <span id="span"> 
+        <span><img src="media/Ellipse.png" alt="image"></span>
+      </span>
+      <h2>Contact me</h2>
+      <p id="contact-me">
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it
+      </p>
+      <br>
+      <br>
+      <form
+        action="https://formspree.io/f/xknyrjqn" method="POST" id="html-form">
+        <fieldset>
+          <input
+            type="text"
+            name="firstname"
+            id="name"
+            maxlength="30"
+            placeholder="Enter your name"
+            required
+          />
+          <input type="email" id="email" name="email" placeholder="Email" />
```

Comment:
- [ ] As the previous reviewer mentioned, please make the email field required. Currently I can fill in the form without email and successfully submit